nodes/text/change_text_case.py

```diff
@@ -28,4 +29,6 @@ def draw(self, layout):
         layout.prop(self, "caseType", text = "")
 
     def getExecutionCode(self):
+        if self.caseType == "CAPITALIZE_WORDS":
+            return "outText = ' '.join( [t.capitalize() for t in inText.split()])"
```

Comment:
Two things:
- you don't need the quare brackets here for the list comprehension. It works without them.
- let's just use this function: https://docs.python.org/3.5/library/stdtypes.html#str.title